pmaweb/templates/contractor.html

```diff
@@ -36,7 +36,7 @@ <h3>Work conditions</h3>
 <ul>
   <li>The Developer shall publish a weekly blog post about his/her work to the phpMyAdmin community.</li>
   <li>The Developer shall send a monthly invoice and report to the Software Freedom Conservancy for the work done during that month.</li>
-  <li>All work produced by the Developer is to be licensed under "GPLv2 or later".</li>
+  <li>All work produced by the Developer is to be licensed under free software licenses and comply with the developer's certificate of origin where applicable.</li>
```

Comment:
How about "licensed under the appropriate free software licenses" or "under the free software licenses and any developer's certificate of origin as required by the modules/project"

Reply:
I've changed it to your first suggestion, probably better to keep this simple and this does it.